gems/bundled_gems

```diff
@@ -5,5 +5,5 @@ rake 13.0.3 https://github.com/ruby/rake
 test-unit 3.4.0 https://github.com/test-unit/test-unit 3.4.0
 rexml 3.2.4 https://github.com/ruby/rexml
 rss 0.2.9 https://github.com/ruby/rss 0.2.9
-rbs 1.0.6 https://github.com/ruby/rbs
 typeprof 0.12.0 https://github.com/ruby/typeprof
+rbs 1.1.1 https://github.com/ruby/rbs
```

Comment:
Oh no, I don't have any opinion about the order of the lines... I just commented out all other lines, and `git add -p` resulted in the changed line order...